### PR TITLE
Figure renames

### PR DESCRIPTION
**Pull request by hollasch**
See #495. This addresses figures. Another PR to follow for images.

### CODE REVIEW

File: books/RayTracingInOneWeekend.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
 write it to a file. The catch is, there are so many formats. Many of those are complex. I always
 start with a plain text ppm file. Here’s a nice description from Wikipedia:
 
-  ![](../images/img.ppm-example.jpg)
+  ![Figure [ppm]: PPM Example](../images/fig-1.01-ppm.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:72]: i use a numbering and name system myself.  good here on the html

File: books/RayTracingInOneWeekend.html [hunk 2]
```diff
@@ -418,7 +418,7 @@
 can go anywhere on the 3D line. For positive $t$, you get only the parts in front of $\mathbf{A}$,
 and this is what is often called a half-line or ray.
 
-  ![Figure [lerp]: Linear interpolation](../images/fig.lerp.jpg)
+  ![Figure [lerp]: Linear interpolation](../images/fig-1.02-lerp.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:421]: i'm not doing a full contextual cross-reference review but naming system looks good

File: books/RayTracingInOneWeekend.html [hunk 3]
```diff
@@ -473,7 +473,7 @@
 sides to move the ray endpoint across the screen. Note that I do not make the ray direction a unit
 length vector because I think not doing that makes for simpler and slightly faster code.
 
-  ![Figure [cam-geom]: Camera geometry](../images/fig.cam-geom.jpg)
+  ![Figure [cam-geom]: Camera geometry](../images/fig-1.03-cam-geom.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:476]: yep

File: books/RayTracingInOneWeekend.html [hunk 4]
```diff
@@ -611,7 +611,7 @@
 (meaning no real solutions), or zero (meaning one real solution). In graphics, the algebra almost
 always relates very directly to the geometry. What we have is:
 
-  ![Figure [ray-sphere]: Ray-sphere intersection results](../images/fig.ray-sphere.jpg)
+  ![Figure [ray-sphere]: Ray-sphere intersection results](../images/fig-1.04-ray-sphere.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:614]: yep

File: books/RayTracingInOneWeekend.html [hunk 5]
```diff
@@ -676,7 +676,7 @@
 personal preference as are most design decisions like that. For a sphere, the outward normal is in
 the direction of the hit point minus the center:
 
-  ![Figure [surf-normal]: Sphere surface-normal geometry](../images/fig.sphere-normal.jpg)
+  ![Figure [sphere-normal]: Sphere surface-normal geometry](../images/fig-1.05-sphere-normal.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:679]: nice

File: books/RayTracingInOneWeekend.html [hunk 6]
```diff
@@ -886,8 +886,8 @@
 the sphere, the normal will point outward, but if the ray is inside the sphere, the normal will
 point inward.
 
-  ![Figure [normal-directions]: Possible directions for sphere surface-normal geometry
-  ](../images/fig.normal-possibilities.jpg)
+  ![Figure [normal-sides]: Possible directions for sphere surface-normal geometry
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:889]: sides?  direction seems more appropriate for a sphere

File: books/RayTracingInOneWeekend.html [hunk 7]
```diff
@@ -1308,7 +1308,7 @@
 For a given pixel we have several samples within that pixel and send rays through each of the
 samples. The colors of these rays are then averaged:
 
-  ![Figure [pixel-samples]: Pixel samples](../images/fig.pixel-samples.jpg)
+  ![Figure [pixel-samples]: Pixel samples](../images/fig-1.07-pixel-samples.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:1311]: yep

File: books/RayTracingInOneWeekend.html [hunk 8]
```diff
@@ -1452,7 +1452,7 @@
 direction randomized. So, if we send three rays into a crack between two diffuse surfaces they will
 each have different random behavior:
 
-  ![Figure [light-bounce]: Light ray bounces](../images/fig.light-bounce.jpg)
+  ![Figure [light-bounce]: Light ray bounces](../images/fig-1.08-light-bounce.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:1455]: yep

File: books/RayTracingInOneWeekend.html [hunk 9]
```diff
@@ -1476,7 +1476,7 @@
 from the hit point $\mathbf{P}$ to the random point $\mathbf{S}$ (this is the vector
 $(\mathbf{S}-\mathbf{P})$):
 
-  ![Figure [rand-vector]: Generating a random diffuse bounce ray](../images/fig.rand-vector.jpg)
+  ![Figure [rand-vec]: Generating a random diffuse bounce ray](../images/fig-1.09.rand-vec.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:1479]: yep

File: books/RayTracingInOneWeekend.html [hunk 10]
```diff
@@ -1693,7 +1693,7 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [random-unit-vector]: <kbd>[vec3.h]</kbd> The random_unit_vector() function]
 
-  ![Figure [rand-unit-vector]: Generating a random unit vector](../images/fig.rand-unitvector.png)
+  ![Figure [rand-unitvec]: Generating a random unit vector](../images/fig-1.10-rand-unitvec.png)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:1696]: yep

File: books/RayTracingInOneWeekend.html [hunk 11]
```diff
@@ -2006,7 +2006,7 @@
 For smooth metals the ray won’t be randomly scattered. The key math is: how does a ray get
 reflected from a metal mirror? Vector math is our friend here:
 
-  ![Figure [reflection]: Ray reflection](../images/fig.ray-reflect.jpg)
+  ![Figure [reflection]: Ray reflection](../images/fig-1.11-reflection.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2009]: yep

File: books/RayTracingInOneWeekend.html [hunk 12]
```diff
@@ -2141,7 +2141,7 @@
 We can also randomize the reflected direction by using a small sphere and choosing a new endpoint
 for the ray:
 
-  ![Figure [reflect-fuzzy]: Generating fuzzed reflection rays](../images/fig.reflect-fuzzy.jpg)
+  ![Figure [reflect-fuzzy]: Generating fuzzed reflection rays](../images/fig-1.12-reflect-fuzzy.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2144]: yep

File: books/RayTracingInOneWeekend.html [hunk 13]
```diff
@@ -2536,7 +2536,7 @@
 I first keep the rays coming from the origin and heading to the $z = -1$ plane. We could make it the
 $z = -2$ plane, or whatever, as long as we made $h$ a ratio to that distance. Here is our setup:
 
-  ![Figure [cam-view-geom]: Camera viewing geometry](../images/fig.cam-view-geom.jpg)
+  ![Figure [cam-view-geom]: Camera viewing geometry](../images/fig-1.14-cam-view-geom.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2539]: yep

File: books/RayTracingInOneWeekend.html [hunk 14]
```diff
@@ -2224,7 +2224,7 @@
 "eta" and "eta prime") are the refractive indices (typically air = 1.0, glass = 1.3–1.7, diamond =
 2.4). The geometry is:
 
-  ![Figure [ray-refract]: Refracted ray geometry](../images/fig.ray-refract.jpg)
+  ![Figure [refraction]: Ray refraction](../images/fig-1.13-refraction.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2227]: inner voice offering resistance where i want to say yep here.  such is life

File: books/RayTracingInOneWeekend.html [hunk 15]
```diff
@@ -2607,14 +2607,14 @@
 constant, you can still rotate your head around your nose. What we need is a way to specify an “up”
 vector for the camera. This up vector should lie in the plane orthogonal to the view direction.
 
-  ![Figure [cam-look]: Camera view direction](../images/fig.cam-look.jpg)
+  ![Figure [cam-view-dir]: Camera view direction](../images/fig-1.15-cam-view-dir.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2610]: view-dir -> vup?

File: books/RayTracingInOneWeekend.html [hunk 16]
```diff
@@ -2721,7 +2721,7 @@
 it's computed (the image is projected upside down on the film). Graphics people, however, usually
 use a thin lens approximation:
 
-  ![Figure [cam-lens]: Camera lens model](../images/fig.cam-lens.jpg)
+  ![Figure [cam-lens]: Camera lens model](../images/fig-1.17-cam-lens.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2724]: yep

File: books/RayTracingInOneWeekend.html [hunk 17]
```diff
@@ -2607,14 +2607,14 @@
...
-  ![Figure [cam-look]: Camera view direction](../images/fig.cam-look.jpg)
+  ![Figure [cam-view-dir]: Camera view direction](../images/fig-1.15-cam-view-dir.jpg)
 
 We can actually use any up vector we want, and simply project it onto this plane to get an up vector
 for the camera. I use the common convention of naming a “view up” (_vup_) vector. A couple of cross
 products, and we now have a complete orthonormal basis $(u,v,w)$ to describe our camera’s
 orientation.
 
-  ![Figure [cam-up]: Camera view up direction](../images/fig.cam-up.jpg)
+  ![Figure [cam-view-up]: Camera view up direction](../images/fig-1.16-cam-view-up.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2617]: yep

File: books/RayTracingInOneWeekend.html [hunk 18]
```diff
@@ -2731,7 +2731,7 @@
 surface of the lens, and send them toward a virtual film plane, by finding the projection of the
 film on the plane that is in focus (at the distance `focus_dist`).
 
-  ![Figure [cam-film-plane]: Camera focus plane](../images/fig.cam-film-plane.jpg)
+  ![Figure [cam-film-plane]: Camera focus plane](../images/fig-1.18-cam-film-plane.jpg)
```
Comment (dafhi) [inline books/RayTracingInOneWeekend.html:2734]: yep

File: books/RayTracingTheNextWeek.html [hunk 19]
```diff
@@ -401,7 +401,7 @@
 divided a set of objects into two groups, red and blue, and used rectangular bounding volumes, we’d
 have:
 
-  ![Figure [bvol-hierarchy]: Bounding volume hierarchy](../images/fig.bvol-hierarchy.jpg)
+  ![Figure [bvol-hierarchy]: Bounding volume hierarchy](../images/fig-2.01-bvol-hierarchy.jpg)
```
Comment (dafhi) [inline books/RayTracingTheNextWeek.html:404]: yep

File: books/RayTracingTheNextWeek.html [hunk 20]
```diff
@@ -440,7 +440,7 @@
 points between two endpoints, _e.g._, $x$ such that $3 < x < 5$, or more succinctly $x$ in $(3,5)$.
 In 2D, two intervals overlapping makes a 2D AABB (a rectangle):
 
-  ![Figure [2daabb]: 2D axis-aligned bounding box](../images/fig.2daabb.jpg)
+  ![Figure [2d-aabb]: 2D axis-aligned bounding box](../images/fig-2.02-2d-aabb.jpg)
```
Comment (dafhi) [inline books/RayTracingTheNextWeek.html:443]: nice

File: books/RayTracingTheNextWeek.html [hunk 21]
```diff
@@ -449,7 +449,7 @@
 example, again in 2D, this is the ray parameters $t_0$ and $t_1$. (If the ray is parallel to the
 plane those will be undefined.)
 
-  ![Figure [ray-slab]: Ray-slab intersection](../images/fig.ray-slab.jpg)
+  ![Figure [ray-slab]: Ray-slab intersection](../images/fig-2.03-ray-slab.jpg)
```
Comment (dafhi) [inline books/RayTracingTheNextWeek.html:452]: yep

File: books/RayTracingTheNextWeek.html [hunk 22]
```diff
@@ -484,7 +484,8 @@
 The key observation to turn that 1D math into a hit test is that for a hit, the $t$-intervals need
 to overlap. For example, in 2D the green and blue overlapping only happens if there is a hit:
 
-  ![Figure [rstio]: Ray-slab $t$-interval overlap](../images/fig.rstio.jpg)
+  ![Figure [ray-slab-interval]: Ray-slab $t$-interval overlap
```
Comment (dafhi) [inline books/RayTracingTheNextWeek.html:487]: outside my juris..

File: books/RayTracingTheNextWeek.html [hunk 23]
```diff
@@ -1917,7 +1918,7 @@
 First, here is a rectangle in an xy plane. Such a plane is defined by its z value. For example, $z =
 k$. An axis-aligned rectangle is defined by the lines $x=x_0$, $x=x_1$, $y=y_0$, and $y=y_1$.
 
-  ![Figure [ray-rect]: Ray-rectangle intersection](../images/fig.ray-rect.jpg)
+  ![Figure [ray-rect]: Ray-rectangle intersection](../images/fig-2.05-ray-rect.jpg)
```
Comment (dafhi) [inline books/RayTracingTheNextWeek.html:1921]: yep

File: books/RayTracingTheNextWeek.html [hunk 24]
```diff
@@ -2397,7 +2398,7 @@
 First, let’s rotate by theta about the z-axis. That will be changing only x and y, and in ways that
 don’t depend on z.
 
-  ![Figure [rotz]: Rotation about the Z axis](../images/fig.rotz.jpg)
+  ![Figure [rot-z]: Rotation about the Z axis](../images/fig-2.07-rot-z.jpg)
```
Comment (dafhi) [inline books/RayTracingTheNextWeek.html:2401]: yep

File: books/RayTracingTheNextWeek.html [hunk 25]
```diff
@@ -2578,7 +2579,7 @@
 far the ray has to travel through the volume also determines how likely it is for the ray to make it
 through.
 
-  ![Figure [ray-vol]: Ray-volume interaction](../images/fig.ray-vol.jpg)
+  ![Figure [ray-vol]: Ray-volume interaction](../images/fig-2.08-ray-vol.jpg)
```
Comment (dafhi) [inline books/RayTracingTheNextWeek.html:2582]: yep

File: books/RayTracingTheRestOfYourLife.html [hunk 26]
```diff
@@ -52,7 +52,8 @@
 problem being a classic case study. We’ll do a variation inspired by that. Suppose you have a circle
 inscribed inside a square:
 
-  ![Figure [circ-sq]: Estimating π with a circle inside a square](../images/fig.circ-sq.jpg)
+  ![Figure [circ-square]: Estimating π with a circle inside a square
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:55]: good

File: books/RayTracingTheRestOfYourLife.html [hunk 27]
```diff
@@ -139,7 +140,7 @@
 We can mitigate this diminishing return by *stratifying* the samples (often called *jittering*),
 where instead of taking random samples, we take a grid and take one sample within each:
 
-  ![Figure [jitter]: Sampling areas with jittered points](../images/fig.jitter.jpg)
+  ![Figure [jitter]: Sampling areas with jittered points](../images/fig-3.02-jitter.jpg)
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:143]: yep

File: books/RayTracingTheRestOfYourLife.html [hunk 28]
```diff
@@ -270,7 +271,7 @@
 First, what is a _density function_? It’s just a continuous form of a histogram. Here’s an example
 from the histogram Wikipedia page:
 
-  ![Figure [histogram]: Histogram example](../images/fig.histogram.jpg)
+  ![Figure [histogram]: Histogram example](../images/fig-3.03-histogram.jpg)
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:274]: yep

File: books/RayTracingTheRestOfYourLife.html [hunk 29]
```diff
@@ -303,7 +304,7 @@
 between 0 and 2 whose probability is proportional to itself: $r$. We would expect the PDF $p(r)$ to
 look something like the figure below, but how high should it be?
 
-  ![Figure [linear-pdf]: A linear PDF](../images/fig.linear-pdf.jpg)
+  ![Figure [linear-pdf]: A linear PDF](../images/fig-3.04-linear-pdf.jpg)
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:307]: yep

File: books/RayTracingTheRestOfYourLife.html [hunk 30]
```diff
@@ -605,7 +606,8 @@
 goes through. The solid angle $\omega$ and the projected area $A$ on the unit sphere are the same
 thing.
 
-  ![Figure [solid-angle]: Solid angle / projected area of a sphere](../images/fig.solid-angle.jpg)
+  ![Figure [solid-angle]: Solid angle / projected area of a sphere
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:609]: yep

File: books/RayTracingTheRestOfYourLife.html [hunk 31]
```diff
@@ -1055,7 +1057,8 @@
 <div class='together'>
 And plot them for free on plot.ly (a great site with 3D scatterplot support):
 
-  ![Figure [pt-uni-sphere]: Random points on the unit sphere](../images/fig.pt-uni-sphere.jpg)
+  ![Figure [rand-pts-sphere]: Random points on the unit sphere
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:1060]: dont know context.  on or in sphere may be helpful

File: books/RayTracingTheRestOfYourLife.html [hunk 32]
```diff
@@ -1336,7 +1339,8 @@
 is $\frac{1}{A}$. What is it on the area of the unit sphere that defines directions? Fortunately,
 there is a simple correspondence, as outlined in the diagram:
 
-  ![Figure [light-pdf]: Projection of light shape onto PDF](../images/fig.light-pdf.jpg)
+  ![Figure [shape-onto-pdf]: Projection of light shape onto PDF
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:1342]: yep

File: books/RayTracingTheRestOfYourLife.html [hunk 33]
```diff
@@ -2196,7 +2200,8 @@
 <div class='together'>
 Now what is $\theta_{max}$?
 
-  ![Figure [sphere-cone]: A sphere enclosing cone](../images/fig.sphere-cone.jpg)
+  ![Figure [sphere-enclosing-cone]: A sphere-enclosing cone
+  ](../images/fig-3.08-sphere-enclosing-cone.jpg)
```
Comment (dafhi) [inline books/RayTracingTheRestOfYourLife.html:2204]: yep

**Comment by hollasch** [comment]
@dafhi — please don't add +1 review comments on a per-change basis. Authors then have to go and resolve these one-by-one.

**Pull request merged**